apps/sms/js/thread_list_ui.js

```diff
@@ -407,14 +474,21 @@ var ThreadListUI = {
   // Since removeThread will revoke list photoUrl at the end of deletion,
   // please make sure url will also be revoked if new delete api remove threads
   // without calling removeThread in the future.
-  delete: function thlui_delete() {
-    function performDeletion() {
-      /* jshint validthis: true */
+  // count = 1 in case of long press on thread 
+  delete: function thlui_delete(selected) {
+    var count = 1;
+
+    if(this.selectionHandler) {
+      selected = this.selectionHandler.selectedList;
+      count = this.selectionHandler.selectedCount;
+    }
+
+    function performDeletion(selected) {
```

Comment:
I don't understand why you need to pass `selected` as argument? Can't you just use the external variable ?